include/mbedtls/config.h

```diff
@@ -232,27 +232,27 @@
 /**
  * \def MBEDTLS_DEPRECATED_WARNING
  *
- * Mark deprecated functions so that they generate a warning if used.
- * Functions deprecated in one version will usually be removed in the next
- * version. You can enable this to help you prepare the transition to a new
- * major version by making sure your code is not using these functions.
+ * Mark deprecated functions and features so that they generate a warning if
```

Comment:
What about macros? For example, we've deprecated the dubious RFC 5114 DHM primes, but that's neither a function deprecation nor a feature deprecation.

Reply:
I personally do not mind saying functions, features and macros if you prefer, but I am not sure this is necessary. I think the term 'feature' is sufficiently broad to tell users that the macros associated with that feature will probably be removed as well.